scripts/generate-app-config/importSideEffects.js

```diff
@@ -6,10 +6,7 @@ const ImportAggregator = require('./generation-objects/ImportAggregator');
 * Resolve relative paths that start with '.', then glob paths, then apply side effect imports
 */
 const importSideEffects = (sideEffects, imports) => {
-  sideEffects.map(sideEffectGlob =>
-    (sideEffectGlob.charAt(0) === '.' ? path.resolve('dev-site-config', sideEffectGlob) : sideEffectGlob)).reduce((acc, sideEffectGlob) =>
-    acc.concat(glob.sync(`${sideEffectGlob}?(.*)`)), []).forEach(sideEffectPath =>
-    imports.addImport(ImportAggregator.relativePath(sideEffectPath)));
+  sideEffects.map(sideEffectGlob => (sideEffectGlob.charAt(0) === '.' ? path.resolve('dev-site-config', sideEffectGlob) : sideEffectGlob)).reduce((acc, sideEffectGlob) => acc.concat(glob.sync(`${sideEffectGlob}?(.*)`)), []).forEach(sideEffectPath => imports.addImport(ImportAggregator.relativePath(sideEffectPath)));
```

Comment:
![](https://media.giphy.com/media/rApKuVWCZZgvm/giphy.gif)

Reply:
Should we maybe disable this section?

Reply:
Yeah I can go update and disable for this bit of code.
I find it more readable the way it was before too.